tests/test_workspaces/test_apis/test_export_settings/test_triggers.py

```
@@ -96,3 +98,131 @@ def test_post_save_workspace_general_settings_export_trigger_3(mocker, db):
+
+    export_trigger = ExportSettingsTrigger(
+        workspace_general_settings=workspace_general_settings_payload,
+        workspace_id=workspace_id,
+        old_configurations={'expense_group_settings': old_expense_group_settings}
+    )
+    export_trigger.post_save_expense_group_settings(new_expense_group_settings)
+
+    assert mock_publish.call_count == 1
+
+
+def test_run_pre_save_expense_group_setting_triggers_no_configuration(db, mocker):
```

Ruff (0.14.0)

Unused function argument: `db`

(ARG001)